src/main/webapp/js/finscholar/errorpageview/errorpageview.js

```diff
@@ -16,29 +16,43 @@
 
 goog.module('finscholar.errorpageview');
 
-const GoogDom = goog.require('goog.dom');
+const {BasicView} = goog.require('basicview');
 const {errorpage} = goog.require('finscholar.errorpageview.templates');
 
+/**
+ * @typedef {{
+ *   occurrence: string,
+ *   action: string,
+ *   errorMessage: string
+ * }}
+ */
+let ErrorData;
+
 /** Class for the error page view. */
-class ErrorPageView {
-  constructor() {}
+class ErrorPageView extends BasicView {
+  /**
+   * @param {!ErrorData=} data Object containing error info.
+   */
+  constructor(data) {
+    super();
+
+    /** @private @type {!ErrorData|undefined} */
+    this.data_ = data;
+
+    const html = errorpage(/** @type {!ErrorData} */ (this.data_));
+    super.setCurrentContent(html);
```

Comment:
Same comment as the last PR, this doesn't belong in the constructor and should be generated dynamically by the render method. I won't make you fix it now because you're short on time, but please address this after your MVP is done.